### PR TITLE
Update and rename bitcoin.7.md to bitcoin-whitepaper.md

### PR DESCRIPTION
**Pull request by itsFrankenSense**
Formatted to match whitepaper.
- Title / Contact details in order
- Added numbers to sections
- Finished missing sections

### CODE REVIEW

**Comment by ibz** [comment]
Amazing! I got caught up with other things and didn't finish. Thank you! It's amazing to have this complete on GitHub!

File: bitcoin-whitepaper.md [hunk 1]
```diff
@@ -165,9 +161,227 @@ Once the latest transaction in a coin is buried under enough blocks, the spent t
...
+     ┌──────────────────────┐
+     │ Transaction          │
+     │                      │
+     │   ┌─────┐  ┌─────┐   │
+─────┼──►│ in  │  │ out │ ──┼─────►
+     │   └─────┘  └─────┘   │
+     │                      │
+     │                      │
+     │   ┌─────┐  ┌─────┐   │
+─────┼──►│ in  │  │ out │ ──┼─────►
```
Comment (ibz) [inline bitcoin-whitepaper.md]: "out" should be "..."

**Comment by itsFrankenSense** [comment]
No worries! Was looking for a markdown version today, and this was the best I came upon. Glad I could help.

File: bitcoin-whitepaper.md [hunk 2]
```diff
@@ -165,9 +161,227 @@ Once the latest transaction in a coin is buried under enough blocks, the spent t
...
+┌──────────────┐ │ ┌──────────────┐    │        │
+│  Identities  │ │ │ Transactions ├───►│ Public │
+└──────────────┘ │ └──────────────┘    │        │
+                                       └────────┘
+```
+As an additional firewall, a new key pair should be used for each transaction to keep them from being linked to a common owner. Some linking is still unavoidable with multi-input transactions, which necessarily reveal that their inputs were owned by the same owner. The risk is that if the owner of a key is revealed, linking could reveal other transactions that belonged to the same owner.
+
+## 11. Calculations
+We consider the scenario of an attacker trying to generate an alternate chain faster than the honest chain. Even if this is accomplished, it does not throw the system open to arbitrary changes, such as creating value out of thin air or taking money that never belonged to the attacker. Nodes are not going to accept an invalid transaction as payment, and honest nodes will never accept a block containing them. An attacker can only try to change one of his own transactions to take back
+money he recently spent.
```
Comment (ibz) [inline bitcoin-whitepaper.md]: No newline needed.

**Review by ibz — CHANGES_REQUESTED** [review]

**Pull request merged**